pywbem/_logging.py

```diff
@@ -14,404 +14,341 @@
 
-            * ``log_comp``: Must be one of strings in the
-              :data:`~pywbem._logging.LOG_COMPONENTS` list.
-
-            * ``detail_level``: Must be one of strings in the
-              :data:`~pywbem._logging.LOG_DETAIL_LEVELS` list.
-
-            * ``dest``: Must be one of strings in the
-              :data:`~pywbem._logging.LOG_DESTINATIONS` list.
+      log_filename (:term:`string`)
+        Optional string that defines the filename for output of logs
+        if the dest type is `file`
 
-          log_filename (:term:`string`)
-            Optional string that defines the filename for output of logs
-            if the dest type is `file`
-
-        Raises:
-
-          ValueError: Generated if the syntax of the input string is invalid
-            or any of the components is not one of allowed strings.
-
-        Examples::
-
-            ops=stderr:min    # set cim operations logger
-
-            http=file:        # set http logger to send to file
-
-            all=file:all      # Set all loggers to default log destination
-        """  # noqa: E501
-        # pylint: enable=line-too-long
-
-        results = cls._parse_log_specs(input_str)
-        for name, value in six.iteritems(results):
-            cls.create_logger(name, log_dest=value[0],
-                              log_detail_level=value[1],
-                              log_filename=log_filename)
-
-    @classmethod
-    def create_logger(cls, log_component, log_dest=DEFAULT_LOG_DESTINATION,
-                      log_filename=DEFAULT_LOG_DESTINATION,
-                      log_detail_level=DEFAULT_LOG_DETAIL_LEVEL):
-        """
-        Create the logger defined by the input parameters and place the result
-        in a class level dictionary in this class.
-
-        This function can be used to set up all of the named loggers used by
-        pywbem.
-
-        Parameters:
+    Raises:
 
-          log_component (:term:`string`):
-            The name of the logger. It must be one of the
-            names defined in :data:`~pywbem._logging.LOG_COMPONENTS`.
-            Used to create the logger name by prepending with the logger name
-            prefix ``pywbem.``.
+      ValueError: Generated if the syntax of the input string is invalid.
 
-          log_dest (:term:`string`):
-            String defining the destination for this log. It must be one of the
-            destinations defined in :data:`~pywbem._logging.LOG_DESTINATIONS`
-            or `None`. If the value is the string ``none``, the null logger is
-            created.
+    Examples::
 
-          log_filename (:term:`string`):
-            Filename to use as logging file if the log destination is `file`.
-            Ignored if log destination is not `file`. If value is `None` and
-            this is a ``file`` log, ValueError is raised.
+        api=stderr:summary    # set cim operations logger for summary display
```

Comment:
* Reference the logger by full name, i.e. "pywbem.api" logger, and not "cim operations logger".
* Mention the destination in the explanation.

Reply:
DONE